### PR TITLE
Fix #478 and other stuff I'd already worked on

### LINKED ISSUE #478: Unable to create New User in 9.2.1 / 9.2.2 on Linux
**Describe the bug**
When installing libation 9.2.1 / 9.2.2 and clicking "New User", the program crashes without output. 
it works fine in 9.2 and below. 

**To Reproduce**

1. Install Libation
2. Open programs
3. Click New User

**Expected behavior**
The window to create new settings should appear.

**Platform**

Linux pop os 22.04 

**Log Files**
Attach your Libation log file here.

### PR DESCRIPTION
**Pull request by Mbucari**
The main reason for this PR is to fix #478. The rest is stuff I'd already completed but hadn't yet gotten around to requesting a merge. But it is ready.

The new "feature" is improved conversion ETA estimates accuracy.
I also did a bit more refactoring of naming templates, and I moved some UI code shared by classic and chardonnay into AppScaffolding.

### CODE REVIEW

**Comment by rmcrackan** [comment]
> I moved some UI code shared by classic and chardonnay into AppScaffolding

I'd like to keep AppScaffolding as agnostic as possible regarding UI, CLI, and Hangover. What if we create a LibationUiBase.

```
AppScaffolding
├── LibationCli
├── LibationUiBase
│   ├── LibationAvalonia
│   └── LibationWinForms
└── HangoverBase
    ├── HangoverAvalonia
    └── HangoverWinForms
```

**Comment by Mbucari** [comment]
> What if we create a LibationUiBase.

Good idea. In the future I'll try to comb through classic and chardonnay and move shared code into LibationUiBase.

File: Source/FileLiberator/_InternalsVisible.cs [hunk 1]
```diff
@@ -1 +0,0 @@
-﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleTo(nameof(FileLiberator) + ".Tests")]
```
Comment (rmcrackan) [inline Source/FileLiberator/_InternalsVisible.cs:1]: Are tests no longer hitting protected items?

File: Source/AaxDecrypter/AverageSpeed.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
... 1.156569922, 1.156512543, 1.156456213, 1.156400904, 1.156346587, 1.156293237, 1.156240827, 1.156189334, 1.156138733, 1.156089001, 1.156040117, 1.155992058, 1.155944804, 1.155898335, 1.155852631, 1.155807674, 1.155763446, 1.155719928, 1.155677105, 1.155634959, 1.155593475, 1.155552637, 1.15551243, 1.155472839, 1.155433851, 1.155395452, 1.155357629, 1.155320368, 1.155283658, 1.155247486, 1.155211841, 1.15517671, 1.155142084, 1.15510795, 1.1550743, 1.155041122, 1.155008406, 1.154976144, 1.154944326, 1.154912942, 1.154881984, 1.154851443, 1.154821311, 1.15479158, 1.154762241, 1.154733287, 1.154704711, 1.154676505, 1.154648662, 1.154621175, 1.154594037, 1.154567242, 1.154540783, 1.154514654, 1.154488849, 1.154463361, 1.154438185, 1.154413316, 1.154388747, 1.154364474, 1.15434049, 1.154316792, 1.154293373, 1.154270229, 1.154247355, 1.154224746, 1.154202398, 1.154180307, 1.154158467, 1.154136875, 1.154115526, 1.154094417, 1.154073543, 1.1540529, 1.154032485, 1.154012294, 1.153992323, 1.153972568, 1.153953027, 1.153933695, 1.15391457, 1.153895647, 1.153876925, 1.153858399, 1.153840066, 1.153821925, 1.15380397, 1.153786201, 1.153768613, 1.153751204, 1.153733972, 1.153716914, 1.153700026 };
```
[the remark this answers is not in the document]
Comment (Mbucari) [reply Source/AaxDecrypter/AverageSpeed.cs:70]: Stats tables for Student's t-Distribution lol

File: Source/FileLiberator/_InternalsVisible.cs (hunk 1 above)
Comment (Mbucari) [reply Source/FileLiberator/_InternalsVisible.cs:1]: Nope. IDK when that changed, but it did.

**Comment by rmcrackan** [comment]
> LibationUiBase

Thanks. I did a slight re-org in the solution to group these apps. It's just virtual folders so it shouldn't affect anything else.

**Pull request merged**